### ISSUE TITLE
Support a URL to the custom.yml file as a parameter to the mdps_generate script

### ISSUE THREAD

**Issue by gschmutz** [TrivadisPF/platys#29]
At the moment the`mdps_generate`script assumes that the 1st parameter references the `custom.yml` using an absolute path. 
For automatic, script deployment scenarios it would nice, if an URL could be passed as well. By that the config.yml could be held anywhere on the internet (i.e. as a simple gist) and would be downloaded by the `mdps_generate` script or inside the docker container which is run by the script.

This would be helpful to setup a simple stack on Lightsail.

**Comment by gschmutz** [comment]
I think it's better to move away from positional parameter in the `mdps-generate.sh` script and use arguments instead. By that we can easily support optional arguments.

this could be a first cut, with the -cr used to pass the URL to the custom.yml:
 
```
mdp-generate.sh -cf <local-custom-config-file> -cr <uri-to-custom-config-file> -o <output-folder> -v <version-of-generator>
mdp-generate.sh --config-file <local-custom-config-file> --config-uri <uri-to-custom-config-file>
--output <output-folder> --version <version-of-generator>
```

**Comment by gschmutz** [comment]
Have implemented a first basic implementation of this item. It was not too hard once I had the named arguments. 
The `--debug` option needs some more love.

**Comment by ufasoli** [comment]
Implemented by @gschmutz on 12.01
